From 6dd31db6159014f23166f48853dfe2a9b52c754b Mon Sep 17 00:00:00 2001
From: Jason Song <sxj@rock-chips.com>
Date: Thu, 19 Nov 2020 14:46:30 +0800
Subject: [PATCH] arm64: configs: rockchip_defconfig: enable DIO5632

This is needed by rk3566 rkg11 2k tablet LCD power supply.

Signed-off-by: Jason Song <sxj@rock-chips.com>
Change-Id: I0a752a92f62b4e72e8778a62ffaabf6fb3cf793f
---
 arch/arm64/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index dce934a81f28..c767446ff8fc 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -543,6 +543,7 @@ CONFIG_REGULATOR_PWM=y
 CONFIG_REGULATOR_RK808=y
 CONFIG_REGULATOR_TPS65132=y
 CONFIG_REGULATOR_XZ3216=y
+CONFIG_REGULATOR_DIO5632=y
 CONFIG_MEDIA_SUPPORT=y
 CONFIG_MEDIA_CAMERA_SUPPORT=y
 CONFIG_MEDIA_CEC_SUPPORT=y
-- 
2.35.3

